filters.txt

```diff
@@ -53,6 +53,9 @@
 _clickthru.swf?
 @@/moje-inzeraty/
 @@/moje-konto/inzeraty/
+@@https://games.tiscali.cz/img/bg/sprite.png
+@@https://games.tiscali.cz/img/bg/main.jpg
+@@https://games.tiscali.cz/img/ico/instagram_icon.svg
```

Comment:
Tahle ikona je u me blokovana pomoci jinych filtru a podle me to sem tedy nepatri:

Fanboy’s Annoyance List 
Fanboy’s Social Blocking List

Reply:
Pravda. Vyhozeno.